src/bodywork/cli/secrets.py

```diff
@@ -140,12 +140,21 @@ def display_secrets(
     if secret_name and not group:
         print_warn("Please specify which secrets group the secret belongs to.")
         return None
-    elif secret_name and group:
+    if secret_name:
```

Comment:
I believe that `elif secret_name and group` means you don't need to cast sh*t as `str` later on, as your friend Mypy understand my version (which is why it is, the way it is) 🙂

Reply:
Mypy forcing us to right less optimal code, i'm not cool with this. That means we need to do make a nested `if` which isn't as nice.

Reply:
I meant use,

`if secret_name and group:`

Instead of,

`if secret_name:`

I suggest we come up with a 'Mypy strategy' after v3. I am in agreement now - as it is, it's annoying.

Reply:
Sorry I should've explained with example above. This means we have an extra check for `group` because the following statement is `elif group` however it's probably better to refactor with the nested `if` to remove checking `group` three times:
```
    if secret_name:
        if not group:
            print_warn("Please specify which secrets group the secret belongs to.")
            return None
        else:
            try:
                complete_secret_name = _create_complete_secret_name(
                    str(group), str(secret_name)
                )
                print_dict(
                    secrets[complete_secret_name].data,
                    _create_table_name(secret_name, group),
                )
            except KeyError:
                print_warn(f"Cannot find secret={secret_name}.")
    elif group:
```

Either way Mypy has forced us into writing code that isn't as nice. My vote is still to disable Mypy for this but the above is the best compromise if we have to conform to Mypy.